makerdao/oracles-v2
# Add multiple data sources and transports capabilities

### Manual testing procedure
```shell
vagrant up
vagrant ssh
oracle install commit <full commit id>
oracle configure
oracle enable
oracle log
```
> see `vagrant/oracle.sh` file for how it works

As a side effect it solves "Adding Gofer to Omnia Configuration [ch10820]"
